.env-test

```diff
@@ -0,0 +1 @@
+PGDATABASE=test-data
```

Comment:
- as per other .env file

Reply:
I have fixed the file naming convention with this .env file now